product-list/index.js

```diff
@@ -0,0 +1,32 @@
+const fastify = require('fastify')({
+  logger: true
+})
+
+fastify.get('/', async (request, reply) => {
```

Comment:
You dont need this endpoint. Can you remove it please?

Reply:
Cool stuff